### ISSUE TITLE
how can I add project reference output to sub folder

### ISSUE THREAD

**Issue by srxqds** [dotnet/runtime#63036]
we want to add project reference and save it to sub folder like ./Libs

but it is not loaded auto

**Comment by Symbai** [comment]
Probably related (in terms of customizing the output layout, moving DLLs into sub folder etc) #3453

**Comment by srxqds** [comment]
I config `additionalProbingPaths` in `runtimeconfig.json` not work.

**Comment by Symbai** [comment]
Yes, that's what the issue is about. If that is exactly what you want you should close this one because its a duplicate and upvote + follow the existing tracking issue.

**Comment by agocke** [comment]
Dup of https://github.com/dotnet/runtime/issues/3453
